### ISSUE TITLE
Stop deleting packages when trimming repo

### ISSUE THREAD

**Issue by ermo** [getsolus/ferryd#24]
People should still be able to roll back if needed.

**Comment by sheepman4267** [comment]
Agreed, with the possible exception of packages deprecated for licensing reasons. In that case, we should likely make every possible effort to make sure we're not distributing those binaries.
